src/loans/index.js

```diff
@@ -39,6 +39,7 @@ function loanTotals( data ) {
   data.federalTotal = data.perkins +
                       data.staffSubsidized +
                       data.staffUnsubsidized +
+                      data.directPlus +
```

Comment:
as I read it, "directPlus" loans encompass Plus loans to graduates and Plus loans to parents.
so here, is data.directPlus just representing parental Plus loans? If so, should we call it that?